### PR TITLE
Desktop: Add duplicate Line command

### PR DESCRIPTION
**Pull request by CalebJohn**
Follow-up to #4869

### CODE REVIEW

**Review by tessus — COMMENTED** [review]
LGTM.

I've noticed that I sometimes miss bein able to join (multiple) lines and I think it's a useful feature of an editor. (Yes, Joplin is not a full fledged editor. I get that.)
We don't have to assign a default shortcut, but make it at least available.

What do you think?

File: packages/app-desktop/gui/NoteEditor/commands/editorCommandDeclarations.ts [hunk 1]
```diff
@@ -87,6 +87,10 @@ const declarations: CommandDeclaration[] = [
 		name: 'editor.deleteLine',
 		label: () => _('Delete line'),
 	},
+	{
+		name: 'editor.duplicateLine',
+		label: () => _('Duplicate Line'),
```
Comment (laurent22) [inline packages/app-desktop/gui/NoteEditor/commands/editorCommandDeclarations.ts:92]: It's a minor thing but for consistency it should be "Duplicate line". We aren't really consistent across the app but as a general rule we should use British capitalisation, which means only the first word is capitalised.

**Review by laurent22 — COMMENTED** [review]
Thanks for the update. All looks good except a minor issue with capitalisation.

File: packages/app-desktop/gui/NoteEditor/commands/editorCommandDeclarations.ts (hunk 1 above)
Comment (tessus) [reply packages/app-desktop/gui/NoteEditor/commands/editorCommandDeclarations.ts]: fixed.

Comment (CalebJohn) [reply packages/app-desktop/gui/NoteEditor/commands/editorCommandDeclarations.ts]: Thanks @tessus

Comment (tessus) [reply packages/app-desktop/gui/NoteEditor/commands/editorCommandDeclarations.ts]: Yw. I hope I wasn't stepping on your toes.

Comment (CalebJohn) [reply packages/app-desktop/gui/NoteEditor/commands/editorCommandDeclarations.ts]: Not at all, just appreciated.

**Comment by laurent22** [comment]
Looks good now, thanks!

**Pull request merged**